development/docker-compose.dev.yml

```diff
@@ -32,17 +31,9 @@ services:
     user: node
     volumes:
       - ../nautobot/ui/:/opt/node/nautobot
+      - ../examples/:/opt/examples
```

Comment:
Curious why this is needed? Can we not pull the example plugin in from `/source`?

Reply:
We're using relative paths in our `jsconfig.json` so our path to the example plugin ui looks like `../../examples/ui/`. This is to support the same relative path in the node container.